src/bom/bitcode.rs

```diff
@@ -910,14 +916,19 @@ struct CompileModifiers {
     /// Corresponding to the compile command including -S to generate an assembly file instead of an object file
     is_assemble_only : bool,
     /// Corresponding to the -E preprocess only flag
-    is_pre_proc_only : bool
+    is_pre_proc_only : bool,
+    /// Compiler invocation does not actually generate any code output.  For
+    /// example, "gcc --version".  Note that this is a subset of
+    /// is_non_generative, but kept for analytical reasons.
```

Comment:
This comment is a bit confusing since it seems to be documenting `is_non_generative`. Is it referring to something else?